corehq/apps/oauth_integrations/models.py

```diff
@@ -1,10 +1,44 @@
+from tkinter import CASCADE
```

Comment:
I don't see this used in anyplace in code. Is this imported for some reason?

Reply:
It's used in as the value of on_delete in the schedule field of the LiveGoogleSheetRefreshStatus model

Reply:
Shouldn't it be `models.CASCADE` then?

Reply:
[tkinter](https://docs.python.org/3/library/tkinter.html) is a GUI toolkit for python, I don't think we would be needing it in the model.

Reply:
Yeah I just checked. it's supposed to be models.deletion.CASCADE. I fixed it!

Reply:
I was wondering what is the difference between `models.CASCADE` and `models.deletion.CASCADE`. I haven't used latter one before.

Reply:
I'm not sure there is a difference. I just saw that Jenny has been using `models.deletion.CASCADE` so I assumed that that was the right way

Reply:
Can you please share some snippets where it is used?

Reply:
Oh I think I see the difference. `models.deletion.CASCADE` is used in the migrations themselves. `models.CASCADE` is mostly used in model code itsself. I'll change it